### PR TITLE
styling changes

### PR DESCRIPTION
**Pull request by KillariDev**
fix https://github.com/DarkFlorist/PetalLock/issues/7
image

### CODE REVIEW

**Review by MicahZoltu — APPROVED** [review]

File: app/css/index.css [hunk 1]
```diff
@@ -74,13 +73,12 @@ svg.spinner > circle {
...
 
 body {
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	height: 100vh;
-	margin: 0;
+	min-height: 100vh;
+    display: grid;
+    place-content: center;
```
Comment (jubalm) [inline app/css/index.css:78]: looks like a spacing issue, just recheck manually as not sure if github will preserve this suggestion
```suggestion
	display: grid;
	place-content: center;
```

File: app/css/index.css [hunk 2]
```diff
@@ -93,12 +91,29 @@ body {
 	color: transparent;
 }
 
-.status {
+.status-red {
 	font-size: 24px;
 	text-wrap: balance;
 	text-align: center;
+	color: rgb(180, 60, 66);
+	border: 4px solid rgb(180, 60, 66);
```
Comment (jubalm) [inline app/css/index.css]: just for consistency across (.status-green uses hex), use either hex value or RGB.
```suggestion
	border: 4px solid #b43c42;
```

File: app/css/index.css [hunk 3]
```diff
@@ -93,12 +91,29 @@ body {
...
+	border-radius: 25px;
+	padding: 20px;
+	box-shadow: 0 0 20px rgb(180, 60, 66);
```
Comment (jubalm) [inline app/css/index.css]:
```suggestion
	box-shadow: 0 0 20px #b43c42;
```

File: app/css/index.css [hunk 4]
```diff
@@ -173,15 +188,17 @@ button:disabled, button[disabled] {
...
+	width: 35px;
 	position: relative;
 	background-color: #FE0006;
 	text-align: center;
 	font-weight: 600;
 	color: white;
 	border-radius: 3px;
 	outline: none;
+	font-size: x-large;
+	align-items: center
```
Comment (jubalm) [inline app/css/index.css]: `align-items` only applies to grid or flex items, missing semi-colon and the `display: grid;` declaration

```suggestion
	align-items: center;
	display: grid;
```

File: app/css/index.css [hunk 5]
```diff
@@ -203,3 +220,15 @@ button:disabled, button[disabled] {
...
 }
+
+.requirement {
+	font-size: large;
+}
+
+.form-control {
+	display: grid;
+    grid-template-columns: min-content min-content auto;
+    padding: 10px;
```
Comment (jubalm) [inline app/css/index.css]: mixed indentation
```suggestion
	grid-template-columns: min-content min-content auto;
	padding: 10px;
```

**Review by jubalm — APPROVED** [review]

**Pull request merged**